deployments/rialto/docker-compose.yml

```
@@ -65,6 +65,20 @@ services:
       - bridge-node-bob
       - bridge-node-charlie
 
+  relay-eth-submit-exchange-tx:
+    build:
+      context: .
+      dockerfile: ./Bridge.Dockerfile
+      args:
+        BRIDGE_HASH: master
+        PROJECT: ethereum-poa-relay
+    entrypoint: /config/relay-eth-submit-exchange-tx-entrypoint.sh
+    volumes:
+      - ./bridge-config:/config
+    environment:
+      RUST_LOG: rpc=trace,bridge=trace
```

```suggestion
      RUST_LOG: rpc=trace,bridge=trace
      GENERATOR_MAX_DELAY: ${GENERATOR_MAX_DELAY:-60}
```

Adding this env variable will make it 60s by default, but if ENV variable is present when running `docker-compose up` (or a `.env` file is present) it can be overwritten.
I think it would be cool to have, because for tests we can easily set it to 60, but in production we can send less/more frequently.